proto/netbookai/spawner/spawner.proto

```diff
@@ -506,4 +515,32 @@ message RegisterClusterOIDCRequest {
   string accountName = 3;
   string clusterName = 4;
 }
+
 message RegisterClusterOIDCResponse {}
+
+message Route53Record {
+  string id = 1;
+  string type = 2;
+  string name = 3;
+  string value = 4;
```

Comment:
And this should be an array of `records`

Reply:
this is the type of which records will be, request and response are array of records

Reply:
In AWS HostedZones have `ResourceRecordSets` which have the `name`, `type`, and array `ResourceRecord`. If this object is `ResourceRecord` then it should not have `name`, and `type`

Reply:
I have simplified the Value field here as we discussed specifically for "TXT" records, The Value field will be under ResourceRecords if it's an A record and it will be under AliasTarget if it's a TXT record. We need to handle cases on basis of recordType then

Reply:
Even for TXT it's under ResourceRecords, right?

Reply:
Sorry, yeah for TXT it's under ResourceRecords, AliasTarget seems like an alias to other DNS name, so it depends if the value will be in AliasTarget or under ResourceRecords